charts/consul/test/unit/server-statefulset.bats

```diff
@@ -1785,3 +1785,20 @@ load _helpers
     yq -r '.containers[0].volumeMounts[] | select(.name == "consul-ca-key")' | tee /dev/stderr)
   [ "${actual}" = "" ]
 }
+
+#--------------------------------------------------------------------
+# ui.dashboardURLTemplates.service
+
+@test "server/StatefulSet: ui.dashboardURLTemplates.service sets the template" {
```

Comment:
do we need to add a test for default behavior?

Reply:
done.